src/ypspur_ros.cpp

```diff
@@ -625,55 +628,78 @@ class YpspurRosNode
     {
       if (i > 0 || YP::YPSpur_initex(key_) < 0)
       {
-        ROS_WARN("launching ypspur-coordinator");
-        pid_ = fork();
-        if (pid_ == 0)
+        std::vector<std::string> args;
+        args.push_back(ypspur_bin_);
+        args.push_back(std::string("-d"));
+        args.push_back(port_);
+        args.push_back(std::string("--admask"));
+        args.push_back(ad_mask);
+        args.push_back(std::string("--msq-key"));
+        args.push_back(std::to_string(key_));
+        if (digital_input_enable_)
+          args.push_back(std::string("--enable-get-digital-io"));
+        if (simulate_)
+          args.push_back(std::string("--without-device"));
+        if (param_file_.size() > 0)
         {
-          std::vector<std::string> args;
```

Comment:
Doing string processing after fork causes a kind of dead lock on strict environments like Alpine.